CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 First alpha release.
 
+[See migration guide for details upgrading from 2.x to 3.x](https://github.com/auraphp/Aura.Sql/blob/695b240f1bd64dd5ba457935d7c0337be1055be5/docs/upgrade.md)
```

Comment:
> Having said that, a "final" conversion guide should be part of the 3.0.0 stable release.

Yes, that may be good. I have been looking how to reduce the changes mentioned regarding parser etc. For it is internal. So may be user level changes may be documented.